dj/models/node.py

```diff
@@ -501,6 +492,9 @@ class SourceNodeFields(BaseSQLModel):
     Source node fields that can be changed.
     """
 
+    catalog: Optional[str] = ""
+    schema_: Optional[str] = ""
+    table: Optional[str] = ""
```

Comment:
If these are optional should we make the default `None`?

Reply:
Good idea, updated 👍